common/src/main/java/com/wynntils/core/config/objects/ConfigHolder.java

```diff
@@ -16,7 +16,7 @@ public ConfigHolder(Object parent, Field field, Config metadata) {
                 parent,
                 field,
                 field.getType(),
-                field.getAnnotation(FeatureInfo.class).category(),
```

Comment:
What's the purpose of this? Are you moving the annotation from the field to the class? 🤔

Reply:
the original line was incorrect - it was trying to get the `@FeatureInfo` annotation of the config field, which won't exist because the annotation is on the parent class